Mods/ArchipelagoPlayer.SC2Mod/enUS.SC2Data/LocalizedData/GameHotkeys.txt

```diff
@@ -259,6 +262,7 @@ Button/Hotkey/AP_PoisonNova=X
 Button/Hotkey/AP_Predator=P
 Button/Hotkey/AP_PredatorFury=F
 Button/Hotkey/AP_PrimalHeal=E
+Button/Hotkey/AP_PrimalIgniterMerge=R

```

Comment:
Can't be `R` due to conflict with Burrow:
![Snímek obrazovky pořízený 2024-07-05 11-11-03](https://github.com/Ziktofel/Archipelago-SC2-data/assets/3447808/29a4de27-a1ac-41ca-b217-ff555614b804)
Pick `G` for example